src/main/java/tommi/InputHandler.java

```diff
@@ -0,0 +1,58 @@
+package tommi;
+
+public class InputHandler {
+
+    public static void processInputCases(String input) {
+        try {
+            if (input.equals("list")) {
+                TaskList.listTasks();
+                return;
+            }
+
+            String[] words = input.split(" ");
+            int taskIndex;
+
+            switch (words[0]) {
+            case "mark":
+                taskIndex = Integer.parseInt(words[1]) - 1;
+                TaskList.markTask(taskIndex);
+                break;
+            case "unmark":
+                taskIndex = Integer.parseInt(input.substring(7)) - 1;
```

Comment:
Try to avoid magic literals(e.g. numbers, strings), you can consider using named constants to give the literals a meaning so that it will be easier to understand. 
E.g.
```
static final double PI = 3.14236;
static final int MAX_SIZE = 10;
...
return PI;
...
return MAX_SIZE - 1;

```